pkg/controller/swiftproxy/swiftproxy_controller.go

```diff
@@ -143,7 +143,7 @@ func (r *ReconcileSwiftProxy) Reconcile(request reconcile.Request) (reconcile.Re
 		return reconcile.Result{}, nil
 	}
 	swiftProxy.Status.ClusterIP = svc.Spec.ClusterIP
-	swiftProxy.Status.LoadBalancerIP = svc.Spec.LoadBalancerIP
+	swiftProxy.Status.LoadBalancerIP = getLoadBalancerIP(svc)
```

Comment:
May be we could create this Swift proxy service using our k8s/service package.

Reply:
hmm that is a good idea :)